### PR TITLE
Project 1: Positioning and Floating Elements

### PR DESCRIPTION
**Pull request by george-swift**
This assignment consists of making a clone of a New York Times article page.

### CODE REVIEW

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work, team. 💻 👀 ✨ 

## However, there are some changes required 👀 🧰 🔨 🔧 
- Please add a better description to your Pull Request. Write a list of the new concepts you used and learned in this project, and where you used them here.
- Your page is overflowing to the right. Please fix this:
image
- The Editor's picks isn't really floating to the right of the text. Instead, you only aligned it to the right, while sill being in-line with the center paragraph. Please fix this so that this aside behaves the same as in the original page:

| Your version | Original |
| --- | --- |
| image | image |


### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Comment by george-swift** [comment]
## Project One: Code Review Improvements

The following concepts were used:
- Flexbox for nav, section, and footer elements
- Float to position aside element on the rightmost side of the page
- Grid to display the diagram in the article.
- Changes in width to remove page overflow.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+  content: "";
+  background-color: #fff;
+  border-bottom: 1px solid #e2e2e2;
+  border-top: 2px solid #e2e2e2;
+}
+
+.times-home,
+.sections-wrapper {
+  width: 100%;
+}
```
Comment (andresporras3423) [inline style.css:290]: Group this selector and the selector on line 120

**Review by andresporras3423 — COMMENTED** [review]
### Status: Changes Required :recycle:
**Hello George Itok** 
Good job with the improvements for this project :muscle:  :muscle:
Your page meets most of the requirements :clap: :clap: :clap:  
There are just a couple of details to improve before you can go ahead:

- [ ] Add a screenshot in the README file
- [ ] In your CSS file, group selectors with the same style definition
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Comment by george-swift** [comment]
## Project One: Code Review Improvements #2
The following additions were made:
- Selector on line 120 grouped with selectors on line 287
- Screenshot and added to the README file

**Review by ebeagusamuel — APPROVED** [review]
# Project Approved :100: 
### Congratulations :champagne: 
#### This project is approved, you can go ahead and merge this branch and start the next project.
#### Good luck
slack(@ebeagusamuel)

**Pull request merged**